### PR TITLE
Backup with kopia

### PR DESCRIPTION
**Pull request by delyriand**

### CODE REVIEW

File: backup-kopia.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    kopia snapshot create ${APP_HOME}/${backupPath}
+done;
+
+# Verifying Validity of Snapshots
+kopia snapshot verify
+SERVICE_EXIT_STATUS=$?
+if [ $SERVICE_EXIT_STATUS -ne 0 ];then
+    echo "Please check the validity the Ooprint snapshots and fix them: https://kopia.io/docs/advanced/consistency/#repairing-corruption" \
+        | mail -s "Kopia Snpashots are unvalid" ${ALERT_EMAIL}
```
Comment (jacquesbh) [inline backup-kopia.sh:71]: We need to set MTA envs for this right?

**Review by jacquesbh — APPROVED** [review]

File: backup-kopia.sh (hunk 1 above)
Comment (maximehuran) [inline backup-kopia.sh:71]:
```suggestion
        | mail -s "Kopia Snapshots are unvalid" ${ALERT_EMAIL}
```

File: clevercloud/functions.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -l
+
+set -o errexit -o nounset -o xtrace
+
+function install_dependencies() {
+    mkdir -p ${HOME}/.local/bin
+
+    # Download and extract kopia
+    wget https://github.com/kopia/kopia/releases/download/v0.15.0/kopia-0.15.0-linux-x64.tar.gz
```
Comment (maximehuran) [inline clevercloud/functions.sh:9]:
```suggestion
    wget https://github.com/kopia/kopia/releases/download/v0.15.0/kopia-0.15.0-linux-x64.tar.gz
```


**❔ question:** ‏ We want to lock the version to `0.15.0` ?

File: backup-kopia.sh [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    | gzip > $APP_HOME/db-backups/$alias.$db.$date.sql.gz
+
+    kopia snapshot create $APP_HOME/db-backups
+done;
+
+# Backup filesystems
+env | grep BACKUP_PATH | while read -r line; do
+    read backupPath <<< `echo $line | awk -F= {'print $2'}`
+
+    # @TODO: ignore list
```
Comment (maximehuran) [inline backup-kopia.sh:60]:
```suggestion
    # @TODO: ignore list
```


**❔ question:** ‏Normal to keep this TODO ?

**Review by maximehuran — COMMENTED** [review]
Some questions and one typo

File: backup-kopia.sh (hunk 3 above)
Comment (delyriand) [reply backup-kopia.sh]: yes, for the moment, i am ignoring `cache` but I should be able to customize it.

**Review by maximehuran — APPROVED** [review]
Thanks !

**Pull request merged**